src/components/modals/Settings.tsx

```diff
@@ -119,7 +120,16 @@ const ModalSettings = () => {
       title='Settings'
       actions={({ close }) => (
         <div className={css({ textAlign: 'center' })}>
-          <ActionButton key='close' title='Close' {...fastClick(() => close())} />
+          <ActionButton
+            key='close'
+            title='Close'
+            {...fastClick(() => {
+              close()
+              if (Capacitor.isNativePlatform()) {
+                window.location.reload()
```

Comment:
I think this already refreshes since it calls the modal `close()` function above?

Reply:
Yes, but it calls the ModalComponent's `close` function, which already does the reload.

Reply:
Fixed.